app/services/stories_api/v3/endpoints/story.rb

```diff
@@ -28,12 +28,16 @@ def get
         end
 
         def patch
+          Rails.logger.info 'Stories Issues'
```

Comment:
I'm tempted to keep them until we have these bugs all fixed.  They will be useful for a bunch of other sifter tickets that dan has as well. http://boost.sifterapp.com/issues/7583